scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,14 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val):
+        """ convert float values to float type. keep string as is """
+        try:
+            return float(val)
```

Comment:
@zgalor @twiest IIUC zabbix knows the type of the metrics because you define them there in advance. On the other hand hawkular defines the type at collection-time.
Is it worth to keep this information (the type) in two places? Is there any way to unify it? (I can't think of a way to unify it).